tests/unit/nupic/research/inhibition_object_test.py

```diff
@@ -39,10 +39,11 @@
 
 class InhibitionObjectTest(unittest.TestCase):
 
+  @unittest.skip("Currently fails due to switch from FDRCSpatial2 to SpatialPooler.")
```

Comment:
Please update the skip comment to elaborate on why the switch causes this test to fail (as you said, the new SP doesn't have explicit methods to get inhibition).